src/components/Tabs/OpenEditorsSubMenu.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import { useTabs } from '@/hooks/useTabs';
+import { RemoveTab } from '../Tabs/RemoveTab';
+import { usePathname } from 'next/navigation';
+import Link from 'next/link';
+
+export function OpenEditorsSubMenu() {
```

Comment:
To keep the name convention I'm suggesting, we could rename this to `OpenFilesSubMenu`